keyboards/crkbd/keymaps/cameronjlarsen/features/caps_word.c

```diff
@@ -0,0 +1,164 @@
+// Copyright 2021-2022 Google LLC
```

Comment:
This is a core feature, and this code should be removed. 

https://docs.qmk.fm/#/feature_caps_word

Reply:
I hadn't realized it was implemented into qmk. I have gone ahead and removed the custom implementation for both keymaps.